app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java

```diff
@@ -14,4 +18,11 @@ public static Intent toBrowser(String url) {
         return new Intent(Intent.ACTION_VIEW, uri);
     }
 
+    public static void share(Context context, @NonNull String url) {
+        Intent intent = new Intent(Intent.ACTION_SEND);
+        intent.setType("text/plain");
+        intent.putExtra(Intent.EXTRA_TEXT, url);
+        context.startActivity(Intent.createChooser(intent, context.getString(R.string.share)));
```

Comment:
Sounds good to add a hash tag :+1: 
Which is better `#droidkaigi #droidkaigiA`  or `droidkaigi`?